classes/patterns/class-block-pattern.php

```diff
@@ -43,6 +43,7 @@ public static function get_list(): array {
 			BlankPage::class,
 			DeepDive::class,
 			GetInformed::class,
+			HomePage::class,
```

Comment:
This usage has a capital `P` but [the actual declaration doesn't](https://github.com/greenpeace/planet4-plugin-gutenberg-blocks/pull/884/files#diff-25f0e4e3db0b77616a6f7b7662e425dc8343525cac2a6b8a478461a2ef08b376R20), even if it seems to work as expected we should probably be consistent in the naming?